### PR TITLE
feat(containers): set chunk sizes for zarr

### PR DESCRIPTION
**Pull request by nritsche**

### CODE REVIEW

File: ch_pipeline/processing/daily.py [hunk 1]
```diff
@@ -201,7 +201,10 @@
       params:
         samples: 4096
         save: true
-        output_name: "sstream_{{tag}}.h5"
+        output_name: "sstream_{{tag}}.zarr"
+        output_format: zarr
```
Comment (anjakefala) [inline ch_pipeline/processing/daily.py:205]: The `output_name` suffix should be sufficient to describe the format.

File: ch_pipeline/processing/daily.py [hunk 2]
```diff
@@ -201,7 +201,10 @@
...
+        output_compression: "bitshuffle"
+        output_compression_opts: [0, "lz4"]
```
Comment (anjakefala) [inline ch_pipeline/processing/daily.py:207]: We should set default compression options.

File: ch_pipeline/processing/daily.py [hunk 3]
```diff
@@ -241,8 +242,29 @@
...
+      in: ringmap
+      out: ringmap_truncated
+      params:
         save: true
-        output_name: "ringmap_intercyl_{{tag}}.h5"
+        output_name: "ringmap_{{tag}}.zarr"
+        output_format: zarr
+        output_compression: "bitshuffle"
+        output_compression_opts: [0, "lz4"]
+    - type: draco.core.io.Truncate
```
Comment (anjakefala) [inline ch_pipeline/processing/daily.py:257]: Tristan made changes to `Truncate`, so we should check to make sure this config still holds.

**Comment by ljgray** [comment]
@jrs65 I'm assuming this branch is outdated with the  changes you've since made to zarr, so I'd like to delete it if so

**Pull request closed without merging**